base/src/main/java/com/smartdevicelink/managers/lifecycle/BaseLifecycleManager.java

```diff
@@ -1337,6 +1332,23 @@ void initialize() {
     void saveVehicleType(String address, VehicleType type){
     }
 
+    private void saveVehicleType(List<TransportRecord> activeTransports, VehicleType type) {
```

Comment:
@YaroslavLutsenko I think it should be declared as empty here as well as the overloaded function above.
The specific logic for a BT transport should be overridden in `LifeCycleManager` - this is definitely not a generic logic for all subclasses.

Reply:
@AKalinich-Luxoft Done in (d2ef11f). Please review this changes.